info/v1/container.go

```diff
@@ -306,6 +306,11 @@ type MemoryStats struct {
 	// Units: Bytes.
 	Usage uint64 `json:"usage"`
 
+	// Various statistics reported in memory.stat in cgroupfs. See the section
```

Comment:
All this is just plumbing to get to the information in memory.stat, which is officially documented at https://www.kernel.org/doc/Documentation/cgroups/memory.txt. I don't find it tasteful to manually enumerate the entries here and duplicate the official documentation.

As a compromise, I can limit the scope of the PR to only expose RSS, which is the original request in #975.